draft-ietf-quic-manageability.md

```diff
@@ -124,27 +124,29 @@ QUIC packets may have either a long header, or a short header. The first bit
 of the QUIC header indicates which type of header is present.
 
 The long header exposes more information. It is used during connection
-establishment, including version negotiation, server retry, and 0-RTT data. It
+establishment, including version negotiation, retry, and 0-RTT data. It
 contains a version number, as well as source and destination connection IDs
 for grouping packets belonging to the same flow. The definition and location
 of these fields in the QUIC long header are invariant for future versions of
 QUIC, although future versions of QUIC may provide additional fields in the
 long header {{QUIC-INVARIANTS}}.
 
-Short headers are used after connection establishment. The only information
-they contain for inspection on the path is an optional, variable-length
-destination connection ID.
+Short headers are used after connection establishment, and contain only an
+optional destination connection ID and the spin bit for RTT measurement.
 
-As of draft version 13 of the QUIC transport document, the following
-information may be exposed in QUIC packet headers:
+As of draft version 18 of the QUIC transport document, the following
```

Comment:
Is this style of statement something you want to keep?  You could just say:

> The following information is exposed is QUIC packet headers:

And allow for the possibility of some skew.  That seems increasingly unlikely now though.

Reply:
done, ignoring skew possibility...